src/modules/ArbitraryCrop/info.json

```diff
@@ -0,0 +1,48 @@
+{
+  "name": "arbitrary-crop",
+  "description": "Crops image for any arbitrary quadilateral",
```

Comment:
Maybe add a description about which way these coordinates go. Like top left, bottom right and so on.

Reply:
Will do that!

Reply:
Done!